src/wam2layers/utils/profiling.py

```diff
@@ -78,14 +80,15 @@ def print_progress(self, t, output):
             totals["s_track_upper_restart"] + totals["s_track_lower_restart"]
         )
 
-        # Don't include boundary losses in log messages
-        interior_losses = output.losses[1:-1, 1:-1].sum()
+        interior_losses = area_weighted_output.losses[1:-1, 1:-1].sum()
```

Comment:
Same comment as above, because it's a duplication as already mentioned in line 90:

For periodic boundaries this is not completely correct. Maybe not solve it now, but just add a 'TODO' or note about it at least?